tests/unit/makefile

```diff
@@ -31,7 +31,7 @@ include $(BOUT_TOP)/make.config
 LDFLAGS += -pthread
 
 # Existence of this file indicates that GTEST has been downloaded
-GTEST_SENTINEL = $(BOUT_TOP)/googletest/README.md
+GTEST_SENTINEL = $(BOUT_TOP)/externalpackages/googletest/README.md
```

Comment:
We should probably just change this to use `GTEST_DIR` while we're at it

Reply:
This wouldn't work as `GTEST_DIR = $(BOUT_TOP)/externalpackages/googletest/googletest` -- note the extra `googletest`

Reply:
We could introduce a GTEST_BASE which everything is defined from though.